### PR TITLE
[DEV APPROVED] bump action_plan gem to 4.0.6

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
Details of the changes are given below:

`
    Fix date issue for distance in words dateHelper

      * add Chronic gem
      * add one more spec to cover months with 30 days
      * alter test to cover months with 31 days
      * create sub-contexts to cover month with odd days
      * remove "should" in test
      * move duplicated 'let' to parent context
`

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1429)

### CODE REVIEW

**Comment by nmiddleweek** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
Thanks @nmiddleweek

**Comment by tomas-stefano** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
Thanks @tomas-stefano

**Comment by misaka** [comment]
If this is deployed, it's going to update rio. Can we wait?

**Comment by Tr4pSt3R** [comment]
@misaka nope. It doesn't altar the RIO gem at all.

**Comment by misaka** [comment]
@Tr4pSt3R I didn't say alter the gem, it's going to pull in the latest rio from master, and I've got some changes in there that should go through UAT first.

#1432 fixes this by locking the rio gem to a specific version.

**Comment by misaka** [comment]
@Tr4pSt3R Could you please rebase off of #1432 so that `rio` doesn't get updated?

**Comment by Tr4pSt3R** [comment]
@misaka yup! that's now done

File: Gemfile.lock [hunk 1]
```diff
@@ -309,7 +309,7 @@ GEM
     dotenv (0.10.0)
     dotenv-rails (0.10.0)
       dotenv (= 0.10.0)
-    dough-ruby (5.12.0.267)
+    dough-ruby (5.12.0.269)
```
Comment (misaka) [inline Gemfile.lock]: Do you want to be pulling in a new `dough-ruby`?

Comment (slaywell) [inline Gemfile.lock]: Hey @Tr4pSt3R, this is being handled in my PR - https://github.com/moneyadviceservice/frontend/pull/1430

Comment (Tr4pSt3R) [inline Gemfile.lock]: Hey @slaywell okay I will take it out from this one then

**Comment by Tr4pSt3R** [comment]
@slaywell this is now done! Dough-ruby is no longer being updated by my PR

**Comment by slaywell** [comment]
:+1: for this PR then!

**Comment by jongilbraith** [comment]
Closing this off as long forgotten and no longer needed.

action_plans is at 4.3.0 now.

**Pull request closed without merging**